app/views/budgets/index.html.erb

```diff
@@ -88,7 +88,7 @@
         </div>
 
         <% unless current_budget.informing? %>
-          <div id="map">
+          <div class="map">
```

Comment:
Only because it's was already defined as a class. And to leave the 'id' attribute to be used with specific things in the element. I forgot to change in L5 on `views/budgets/investments/_map.html.erb` too :smile: gonna update it.